src/main/java/org/codinjutsu/tools/jenkins/security/JenkinsVersion.java

```diff
@@ -0,0 +1,20 @@
+package org.codinjutsu.tools.jenkins.security;
+
+/**
+ * Created by bidek on 13.01.17.
+ */
+public enum JenkinsVersion {
```

Comment:
Indeed, it is far more logic that way. I`ll commit patched version by this weekend - i think so